core/src/smartcontracts/wasm.rs

```diff
@@ -1018,59 +1018,59 @@ where
     #[codec::wrap]
     fn set_permission_token_schema(_schema: PermissionTokenSchema, _state: &mut S) {
         panic!(
-            "Validator `{}()` entrypoint should not set permission token schema",
+            "Executor `{}()` entrypoint should not set permission token schema",
             Self::ENTRYPOINT_FN_NAME
         )
     }
 }
 
-impl<'wrld> Runtime<state::validator::ValidateTransaction<'wrld>> {
-    /// Execute `validate_transaction()` entrypoint of the given module of runtime validator
+impl<'wrld> Runtime<state::executor::ValidateTransaction<'wrld>> {
+    /// Execute `validate_transaction()` entrypoint of the given module of runtime executor
     ///
     /// # Errors
     ///
     /// - if failed to instantiate provided `module`
     /// - if unable to find expected function export
     /// - if the execution of the smartcontract fails
-    /// - if unable to decode [`validator::Result`]
-    pub fn execute_validator_validate_transaction(
+    /// - if unable to decode [`executor::Result`]
+    pub fn execute_executor_validate_transaction(
```

Comment:
Maybe make it more readable like:
```suggestion
    pub fn execute_validate_transaction_in_executor(
```
And the same about all the functions with similar names in this module.